src/PKSim.Core/Services/ParameterContainerTask.cs

```diff
@@ -76,11 +76,10 @@ void parameterValueModifier(ParameterMetaData parameterMetaData, IParameter para
          {
             //here we know that the parameter fulfills the criteria above. However, in case of a parameter that is used by all species, we need to verify that either the formula or the value is common 
             //to all species. Otherwise, we set the value to NaN
-            var (isSame, exists) = _sameFormulaOrValueForAllSpeciesRepository.IsSameFormulaOrValue(parameterMetaData);
```

Comment:
I don't thik it could have existedc before and NOT BE THE SAME (since by construction we either had isSameFormula or isSameValue and isSame as A || B
so it could not exist and not be the same. Is this correct?

Reply:
Theoretically yes. We will see 😄